src/analytics/analytics.js

```diff
@@ -319,6 +319,9 @@ export class Analytics {
             regionCard.setLabel('Step ' + this.nextStepNumber);
         }
 
+        let hue = (this.nextStepNumber * 17) % 360;
+        regionCard.setAccentColor(`hsl(${hue}, 100%, 50%)`);
```

Comment:
Another idea might be to derive the color from ordering on timeline rather than current step counter.  In practice, such recoloring happens even now when you reload the viewer.